modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/search/bar/portlet/display/context/factory/SearchBarPortletDisplayContextFactory.java

```diff
@@ -408,6 +415,15 @@ private String _getScopeParameterName(
 		return searchBarPortletPreferences.getScopeParameterName();
 	}
 
+	private SearchRequest _getSearchRequest(
```

Comment:
I think this _getSearchRequest method is to simple to be wanted. Just have 
`SearchResponse searchResponse = portletSharedSearchResponse.getSearchResponse();`  defined at the top.